cmsdb/campaigns/run2_2016_nano_uhh_v12/qcd.py

```diff
@@ -0,0 +1,110 @@
+# coding: utf-8
```

Comment:
As for the comment above, if there are no QCD samples being processed, we should remove this file altogether.

Reply:
removed